Calibration/HcalCalibAlgos/test/gammaJetAnalysis_cfg.py

```diff
@@ -1,13 +1,66 @@
 import FWCore.ParameterSet.Config as cms
 process = cms.Process('ANALYSIS')
 
-process.load('FWCore.MessageService.MessageLogger_cfi')
 process.load('Configuration.StandardSequences.Services_cff')
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
 
-#process.MessageLogger.messages = cms.untracked.PSet(
-#        threshold =  cms.untracked.vstring('DEBUG')
-#)
+process.MessageLogger = cms.Service("MessageLogger",
+    destinations = cms.untracked.vstring('warnings','errors',
+                                         'cout','cerr'),
+    categories = cms.untracked.vstring('GammaJetAnalysis'), 
+    debugModules = cms.untracked.vstring('*'),
+    warnings = cms.untracked.PSet(
+        placeholder = cms.untracked.bool(True)
+    ),
+    default = cms.untracked.PSet(
+
+    ),
+    errors = cms.untracked.PSet(
+        placeholder = cms.untracked.bool(True)
+    ),
+    cerr = cms.untracked.PSet(
+        optionalPSet = cms.untracked.bool(True),
+        INFO = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        noTimeStamps = cms.untracked.bool(False),
+        FwkReport = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            reportEvery = cms.untracked.int32(500),
+            limit = cms.untracked.int32(10000000)
+        ),
+        default = cms.untracked.PSet(
+            limit = cms.untracked.int32(10000000)
+        ),
+        Root_NoDictionary = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            limit = cms.untracked.int32(0)
+        ),
+        FwkJob = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            limit = cms.untracked.int32(0)
+        ),
+        FwkSummary = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            reportEvery = cms.untracked.int32(1),
+            limit = cms.untracked.int32(10000000)
+        ),
+        threshold = cms.untracked.string('INFO')
+     ),
+     cout = cms.untracked.PSet(
+        threshold = cms.untracked.string('INFO'),
+        noTimeStamps = cms.untracked.bool(True),
+        INFO = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        DEBUG = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        GammaJetAnalysis = cms.untracked.PSet(
+            limit = cms.untracked.int32(-1)
+       )
+    )
+)
 
 process.load("Configuration.Geometry.GeometryIdeal_cff")
```

Comment:
@bsunanda 
please change this to `process.load("Configuration.StandardSequences.GeometryRecoDB_cff") as explained [here](https://indico.cern.ch/event/368326/contribution/1/material/slides/0.pdf)